activity-metrics-list.md

```diff
@@ -71,6 +73,7 @@ We list and describe activity metrics. For different situations, the metrics hav
 |Update Regularity|How consistently and frequently are updates provided.|
 |Use of Acronym|*Frequency of acronyms used*\\ Specialized language can be a barrier for new contributors.|
 |User Groups|user groups perform a variety of crucial marketing, service support, and business-development functions at the grassroots level\\ (Bagozzi & Dholakia, 2006)|
+|Velocity|A graph where bubbles’ area is proportional to the number of authors, the y-axis (height) is the total number of pull requests & issues, and the x-axis is the number of commits. [Example](https://www.cncf.io/blog/2017/06/05/30-highest-velocity-open-source-projects/)|

```

Comment:
It seems that this description doesn't really define a metric, but instead describes a visualization of a combination of metrics. Is that OK?

Reply:
Ultimately it is (1) the number of authors against (2) the total number of pull requests & issues against (3) the number of commits

Reply:
Yes, maybe we need a new section for visualizations.

Reply:
Yeah, I guess I'm trying to get a feel for how formal (again, sorry!) we want to be about defining what a metric is.